lib/ui/screen/main.dart

```diff
@@ -305,36 +307,31 @@ class _MyHomePageState extends State<MyHomePage> {
                                     ),
                                     onPressed: () {},
                                   ),
-                                  Text('Home',
+                                  const Text('Home',
                                       style: TextStyle(
                                         fontSize: 20,
                                         fontWeight: FontWeight.normal,
                                       )),
                                 ]),
-                            SizedBox(
+                            const SizedBox(
                               height: 20,
                             ),
                             Expanded(
                               child: AnimatedList(
                                 key: alertProvider.listKey,
                                 initialItemCount: alertProvider.items.length,
                                 itemBuilder: (context, position, animation) {
-                                  if (position < alertProvider.items.length) {
-                                    // last item needs padding for bottom sheet
-                                    if (position ==
-                                        alertProvider.items.length - 1) {
-                                      return Padding(
-                                        padding:
-                                            const EdgeInsets.only(bottom: 80),
-                                        child: buildListItem(context, position,
-                                            animation, alertProvider),
-                                      );
-                                    } else {
-                                      return buildListItem(context, position,
-                                          animation, alertProvider);
-                                    }
+                                  if (position ==
+                                      alertProvider.items.length - 1) {
```

Comment:
this padding causing animation started in lower position lol. I don't see any issue of removing the padding though @WenjiuWang

Reply:
Need this padding here to prevent bottom sheet from blocking the last item 
![Screenshot_20230410224857](https://user-images.githubusercontent.com/56171936/231043055-00dacede-0101-448c-b34e-cb25fd8a1ef5.png)